improve the content related to self signed docs (#687)
aerogear/mobile-docs: modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

certificates used by the back-end services, which are signed by a certificate authority.
Most client devices have a list of well-known and trusted certificate authorities pre-installed and this allows the client devices to verify the certificates used by the back-end services.

> However, this normally doesn't work in a development environment, such as a local OpenShift cluster as it uses self-signed certificates which are not signed by any of the trusted certificate authorities. This means by default the client devices won't be able to establish secure connections with the back-end services that are running on a local OpenShift cluster.

Reviewer: ```suggestion
However, this normally doesn't work in a development environment, such as a local OpenShift cluster as it uses self-signed certificates which are not signed by any of the trusted certificate authorities. As a result, the client devices cannot establish secure connections with the back-end services that are running on a local OpenShift cluster.
```

> To work around the problem, you must manually extract the root certificate from the cluster, install it on the device and make sure your application trusts the new certificate.

Reviewer: ```suggestion
The suggested workaround is to manually extract the root certificate from the cluster, install it on the device, and make sure your application trusts the new certificate.
```

Reviewer: I am manually making this update as part of a new commit.

[tabs]
====

> Android::

Reviewer: @finp / @wei-lee - the (tag::excludeDownstream[]) have been removed from here. I'm not sure if this is in-line with what we are GA'ing in September...it could be, just calling it out for your attention.

+
--
. Emulators

> .. https://support.google.com/android/answer/2819522?hl=en[Set screen lock] on the mobile device to ensure the certificate can be installed.

Reviewer: ```suggestion
.. link:https://support.google.com/android/answer/2819522?hl=en[Set screen lock] on the mobile device to allow for the installation of the certificate.
```

Reviewer: I am manually making this update as part of a new commit.

> .. Drag and drop the certificate file to the emulator, it should be copied to the `Downloads` folder on the device.

Reviewer: ```suggestion
.. Click on the *certificate file* and drag it onto the emulator. It should be copied to the _Downloads_ folder on the device.
```

Reviewer: I am manually making this update as part of a new commit.

.. Install the certificate on your device:
+

> Go to Settings > Security & location > Advanced > Encryption & credentials > Install from SD card, you will be prompted to choose a file. Navigate to the `Downloads` folder and you should see the certificate file. Tap on the certificate file and Android system should detect the

Reviewer: ```suggestion
To choose a file, navigate to _Settings > Security & location > Advanced > Encryption & credentials > Install from SD card_. From here, navigate to the _Downloads_ folder and you should see the certificate file.
```

Reviewer: I am manually making this update as part of a new commit.

> certificate and let you install it. Here is https://support.google.com/nexus/answer/2844832?hl=en[a sample guide for Google Nexus devices].

Reviewer: ```suggestion
To install the certificate, tap on the *certificate file* and Android should detect the certificate. For more information, please see link:https://support.google.com/nexus/answer/2844832?hl=en[a sample guide for Google Nexus devices].
```

. Real Devices

> .. https://support.google.com/android/answer/2819522?hl=en[Set screen lock] on the mobile device to ensure the certificate can be installed.

Reviewer: ```suggestion
.. To allow for the installation of certificates on a mobile device, set the https://support.google.com/android/answer/2819522?hl=en[screen lock].
```

.. Copy the file to your device using one of the following methods:
+
* Email attachment (Recommended) - Simply email the certificate to an address
accessible from the device and download the attachment.
* Cloud service - Use a cloud storage service such as Dropbox or Google
Drive which is accessible from the device, browse to the certificate and
proceed to download it.

> * USB - Attach the device to the machine via USB and drag the certificate to a devices file system. Here is an https://support.google.com/nexus/answer/2840804?hl=en[example guide for Google Nexus devices]. It may be different for other devices. Or, use the link:https://developer.android.com/studio/command-line/adb[Android Debug Bridge^](adb) push command to push the certificate to the connected device:

Reviewer: I think the fix is part of this: https://github.com/aerogear/mobile-docs/pull/688#pullrequestreview-236519038
